corehq/apps/app_manager/static/app_manager/js/xpathConfig.js

```diff
@@ -0,0 +1 @@
+../../../xpath_validator/nodejs/xpathConfig.js
```

Comment:
@benrudolph yeah that's right, this is how a symlink looks in a git diff. As far as I know symlinks are fully supported by git, though it's possible that there's some different behavior on Windows or something; I've never tested that